app/graphql/types/user_type.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Types
+  class UserType < Types::BaseObject
+    field :id, ID, null: false
+    field :first_name, String
+    field :last_name, String
+    field :email, String
+    field :password_digest, String
+    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :updated_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :role, String, null: false
```

Comment:
It is okay, but ideally it should be special type:
```ruby
module Types
  class UserRoleEnum < BaseEnum
    User.role.each_value do |user_role|
      value user_role.to_s.upcase, user_role.text, value: user_role.to_s
    end
  end
end
```

Reply:
@w1nway

Reply:
i think that question was closed